ui/yarn.lock

```diff
@@ -7585,15 +7585,10 @@ domhandler@^4.0.0, domhandler@^4.2.0, domhandler@^4.3.1:
   dependencies:
     domelementtype "^2.2.0"
 
-dompurify@^2.2.0:
-  version "2.2.8"
-  resolved "https://registry.yarnpkg.com/dompurify/-/dompurify-2.2.8.tgz#ce88e395f6d00b6dc53f80d6b2a6fdf5446873c6"
-  integrity sha512-9H0UL59EkDLgY3dUFjLV6IEUaHm5qp3mxSqWw7Yyx4Zhk2Jn2cmLe+CNPP3xy13zl8Bqg+0NehQzkdMoVhGRww==
-
-dompurify@^2.2.8:
-  version "2.3.0"
-  resolved "https://registry.yarnpkg.com/dompurify/-/dompurify-2.3.0.tgz#07bb39515e491588e5756b1d3e8375b5964814e2"
-  integrity sha512-VV5C6Kr53YVHGOBKO/F86OYX6/iLTw2yVSI721gKetxpHCK/V5TaLEf9ODjRgl1KLSWRMY6cUhAbv/c+IUnwQw==
+dompurify@^2.2.0, dompurify@^2.2.8:
```

Comment:
I know nothing of yarn so I'm asking for my own education.  Why does `2.5.7` not go on this line too?  Or why isn't this line replaced by that?

Reply:
IIRC, the versions listed immediately after the package name on the same line are the versions specified as dependencies (or transitive dependencies if they are embedded in another module's `package.json` file), while the version on a separate line with the key `version` is the version that all those listed versions will use.

For background, the caret (`^`) means "this version or any minor or patch version up to but not including the next major version.

So, `^2.2.0` means "any versoin 2.x.y before 3.0.0".

Reply:
Thank you.